monitoring/manager_collector.go

```diff
@@ -0,0 +1,91 @@
+package monitoring
+
+import (
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metricsclientset "k8s.io/metrics/pkg/client/clientset/versioned"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/longhorn/longhorn-manager/datastore"
+)
+
+type ManagerCollector struct {
+	*baseCollector
+
+	kubeMetricsClient *metricsclientset.Clientset
+	namespace         string
+
+	managerCPUUsageMetric    metricInfo
```

Comment:
remove manager prefix same as with the other collectors.

Reply:
Thank you! Fixed